ykman/cli/fido.py

```diff
@@ -74,6 +75,7 @@ def info(ctx):
     """
     Display status of FIDO2 application.
     """
+    ctx.obj['dev'].get()
```

Comment:
This looks like a side effect free expression, which therefore looks useless since it's not assigned to anything. Maybe rename the `get` method to something like `resolve`, or add an alias?